extensions/elasticsearch-java-client/runtime/src/main/java/io/quarkus/elasticsearch/javaclient/runtime/ElasticsearchJavaClientsRuntimeConfig.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.elasticsearch.javaclient.runtime;
+
+import java.util.Map;
+
+import io.quarkus.elasticsearch.restclient.common.runtime.ElasticsearchClientBeanUtil;
+import io.quarkus.runtime.annotations.ConfigDocMapKey;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefaults;
+import io.smallrye.config.WithParentName;
+import io.smallrye.config.WithUnnamedKey;
+
+@ConfigMapping(prefix = "quarkus.elasticsearch-java")
```

Comment:
since we didn't have the config properties for this extension before this ... just pointing your attention to the new prefix for the config group 🙂